R/messages.R

```diff
@@ -0,0 +1,62 @@
+errorWrongType <- function(object, optionalMessage = NULL) {
```

Comment:
We can put all message strings here. We can generate message strings dynamically in functions and then call these functions e.g. when throwing an error.